### DISCUSSION TITLE
What is the latest version? How to export the file that saves the running results to the computer?

### DISCUSSION THREAD

**Discussion by jiahe224** [termux/termux-app#3548] [Q&A] [answered]
The latest version on GitHub is 0.118.0, but this website (https://apkcombo.com/zh/termux/com.termux/) is 0.119.1, is it official or fake? In addition, how to export files to the computer, my file path is /data/data/com.termux/files/home, and the phone is not rooted.

**Comment by twaik** [comment, marked as answer]
> 0.119.1

It is fake. 0.119 is not available publicly, it is still WIP.
The recommended ways to install termux are described here: https://github.com/termux/termux-app#installation

> how to export files to the computer, my file path is /data/data/com.termux/files/home

https://wiki.termux.com/wiki/Internal_and_external_storage. You can copy your files to sdcard which is available to PC via usb connection. 
Also there are more `linux` like ways to do that like `python -m http.server 8080` to open simple web server and let you access your files via a browser.

**Reply by jiahe224** [reply]
Thank you very much.
